src/objects/impls/room_position.rs

```diff
@@ -156,57 +156,57 @@ impl RoomPosition {
     }
 }
 
-impl<T: HasPosition> PartialEq<T> for RoomPosition {
+impl<T: HasPosition> PartialEq<T> for RemoteRoomPosition {
     fn eq(&self, other: &T) -> bool {
         self.local() == other.pos()
     }
 }
 
-impl Eq for RoomPosition {}
+impl Eq for RemoteRoomPosition {}
 
-impl Into<(u8, u8)> for RoomPosition {
+impl Into<(u8, u8)> for RemoteRoomPosition {
     fn into(self) -> (u8, u8) {
         (self.x() as u8, self.y() as u8)
     }
 }
 
-impl Into<(u16, u16)> for RoomPosition {
+impl Into<(u16, u16)> for RemoteRoomPosition {
     fn into(self) -> (u16, u16) {
         (self.x() as u16, self.y() as u16)
```

Comment:
It might be best to make it local once and get `x` and `y` from it. Saves a js-rust transport.

Reply:
Eh- I can change this!

I did the transition commit first mostly to ensure that we can cleanly revert removing `RemoteRoomPosition` if it's deemed necessary in the future, without having to first disambiguate it and `LocalRoomPosition`.